rtic/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ For each category, *Added*, *Changed*, *Fixed* add new entries at the top!
 ### Changed
 
 - Remove unused dependency `rtic-monotonics`
+- Use `riscv-slic` from `crates.io`
```

Comment:
Could you put the newer changelog entries above older ones please? 😇

Awesome with the release 👍

Reply:
Ooops sorry! Done